elide-async/src/main/java/com/yahoo/elide/async/operation/TableExportOperation.java

```diff
@@ -172,13 +170,11 @@ public AsyncAPIResult call() {
     /**
      * Initialize and get RequestScope.
      * @param exportObj TableExport type object.
-     * @param user User Object.
-     * @param apiVersion API Version.
+     * @param scope RequestScope.
```

Comment:
Since this takes a scope and returns a scope - we should add a bit more explanation here in the comments explaining the difference.